src/transformers/feature_extraction_common_utils.py

```diff
@@ -278,12 +251,12 @@ def from_pretrained(
 
 
         Returns:
-            :class:`~transformers.PreTrainedFeatureExtractor`: The feature extractor object instantiated from this
-            pretrained model.
+            :class:`~transformers.PreTrainedSequenceFeatureExtractor`: The feature extractor object instantiated from
+            this pretrained model.
 
         Examples::
 
-            # We can't instantiate directly the base class `PreTrainedFeatureExtractor` so let's show the examples on a
+            # We can't instantiate directly the base class `PreTrainedSequenceFeatureExtractor` so let's show the examples on a
```

Comment:
Examples for ImageFeatureExtractor saving/loading can be appended to the Examples here